runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -266,10 +268,32 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 			case WidgetID.SHOP_INVENTORY_GROUP_ID:
 				menuEntry.setOption(SEARCH_GRAND_EXCHANGE);
 				menuEntry.setType(MenuAction.RUNELITE.getId());
+				entries = Arrays.copyOf(entries, 2);
+				entries[1] = menuEntry;
 				client.setMenuEntries(entries);
 		}
 	}
 
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked event)
```

Comment:
Does this actually work? Because you cant click when holding alt

Reply:
Yes, you release alt.

Reply:
But the menu option is not added when not holding alt, or? At least that is the purpose of the keybind, for it only searching when ALT is pressed

Reply:
The menu still shows up when release alt.

Reply:
That sounds like bug with it not being removed then

Reply:
If you hold alt and click on the menu, it still searches.

Reply:
I cannot even open the right click menu whilst holding alt.

Reply:
it's a very specific timing of right clicking and hitting alt at the same time

Reply:
It is a bug, so I would not add more stuff to make the bug work more, I would just fix the issue you are referencing in this PR rather than doing extra stuff

Reply:
So for anyone to use this feature they have to try ~20 times to get the timing right? This seems stupid, I just managed to get it but this seems wrong.